src/vs/workbench/contrib/notebook/browser/view/renderers/backLayerWebView.ts

```diff
@@ -833,14 +833,23 @@ export class BackLayerWebView<T extends ICommonCellInfo> extends Disposable {
 	}
 
 	async createWebview(): Promise<void> {
+		const baseUrl = this.asWebviewUri(dirname(this.documentUri), undefined);
+
+		// Python hasn't moved to use a preload to load require support yet.
+		// For all other notebooks, we no longer want to include our loader.
+		if (!this.documentUri.path.toLowerCase().endsWith('.ipynb')) {
```

Comment:
Can you have other file extensions? Is there a better check I can do here?

Reply:
Oh I noticed you have an if condition, awesome, when I'm ready I'll submit a PR to remove this in VS Code core.
Thanks again.